icicle/src/ntt/kernel_ntt.cu

```diff
@@ -702,6 +702,14 @@ namespace mxntt {
   {
     CHK_INIT_IF_RETURN();
 
+    if (log_size < 4) {
+      // TODO: should throw exception
+      // TODO: fast twiddles should be used only with mixed radix ntt
+      // THROW_ICICLE_ERR(IcicleError_t::InvalidArgument, "log_size must be >= 4");
+      printf("warning: log_size must be at least 4, got %d\n", log_size); // TODO: logging
```

Comment:
"warning: when using fast twiddles, log_size must be at least 4..

Reply:
umm, throwing would be a breaking change